app/assets/stylesheets/modules/_details.scss

```diff
@@ -0,0 +1,125 @@
+.details-hero {
+  background-color: $white;
+  text-align: center;
+
+  &-title {
+    margin-bottom: em(5);
+  }
+
+  &-compliment {
+    display: inline-block;
+
+    &-refresh {
+      background: image-url("refresh.png");
+      background-repeat: no-repeat;
+      background-size: 14px;
+      display: inline-block;
+      height: em(15);
+      width: em(15);
```

Comment:
Properties should be ordered background, background-repeat, background-size, display, height, margin-left, width